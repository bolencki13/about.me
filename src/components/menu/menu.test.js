import React from 'react'
import renderer from 'react-test-renderer'
import Menu from './'

it('renders without crashing', () => {
  const component = renderer.create(
    <Menu/>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
