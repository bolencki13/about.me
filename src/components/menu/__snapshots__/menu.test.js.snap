// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<nav
  className="navbar navbar-expand-lg"
  id="bto-menu"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      Brian Olencki
    </a>
    <button
      aria-controls="navbarSupportedContent"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarSupportedContent"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarSupportedContent"
    >
      <ul
        className="navbar-nav mr-auto"
      >
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="#projects"
          >
            Projects
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="https://repo.bolencki13.com/"
          >
            Cydia Repo
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
